doc/system_health_monitoring/system-health-HLD.md

```diff
@@ -5,58 +5,53 @@
-For the Monit, now below services and file system is under monitoring:
-
-	admin@sonic# monit summary -B
-	Monit 5.20.0 uptime: 1h 6m
-	 Service Name                     Status                      Type
-	 sonic                            Running                     System
-	 rsyslog                          Running                     Process
-	 telemetry                        Running                     Process
-	 dialout_client                   Running                     Process
-	 syncd                            Running                     Process
-	 orchagent                        Running                     Process
-	 portsyncd                        Running                     Process
-	 neighsyncd                       Running                     Process
-	 vrfmgrd                          Running                     Process
-	 vlanmgrd                         Running                     Process
-	 intfmgrd                         Running                     Process
-	 portmgrd                         Running                     Process
-	 buffermgrd                       Running                     Process
-	 nbrmgrd                          Running                     Process
-	 vxlanmgrd                        Running                     Process
-	 snmpd                            Running                     Process
-	 snmp_subagent                    Running                     Process
-	 sflowmgrd                        Running                     Process
-	 lldpd_monitor                    Running                     Process
-	 lldp_syncd                       Running                     Process
-	 lldpmgrd                         Running                     Process
-	 redis_server                     Running                     Process
-	 zebra                            Running                     Process
-	 fpmsyncd                         Running                     Process
-	 bgpd                             Running                     Process
-	 staticd                          Running                     Process
-	 bgpcfgd                          Running                     Process
-	 root-overlay                     Accessible                  Filesystem
-	 var-log                          Accessible                  Filesystem
-
-
-By default any above services or file systems is not in good status will be considered as fault condition.
-
-### 1.2 Peripheral devices status which could impact the system health status
+System health monitor is intended to monitor both critical services/processes and peripheral device status and leverage system log, system status LED to and CLI command output to indicate the system status.
+
+In current SONiC implementation, monit service can monitor the file system as well as customized check script status, system health monitor can rely on Monit service to monit these items. There are also a set of daemons(psud, thermaltcld, etc.) inside PMON collecting the peripheral devices status.
```

Comment:
`monit` ---> `Monit`?

`customized check script status,` ---> `customized script status.`?

`to monit these items` ---> `to monitor these items`?

`collecting` --> `to collect`?

Reply:
`a set of daemons(psud, thermaltcld, etc.)` ---> `a set of daemons such as psud, thermaltcld`?